tests/suites/sidecar/task.sh

```diff
@@ -0,0 +1,17 @@
+test_sidecar() {
```

Comment:
not in love with "sidecar"
we are fast getting to the point where they will be the only k8s charms
can we rename to test_k8s_charm or something?

Reply:
I don't love `k8s_charm` either, so per discussion, let's merge and worry about grouping/naming as we get more of these.